### PR TITLE
fix ENOTDIR error

### PR DESCRIPTION
**Pull request by sbonasu**
README.md file path is being returned as packagePath which is a file not directory

### CODE REVIEW

**Comment by kellyselden** [comment]
We should have a test for this.

File: src/get-workspaces-paths.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function processGlobs({ cwd, _2dFilesArray, isPnpm }) {
     try {
       packageJson = readJsonSync(path.join(cwd, packagePath, 'package.json'));
     } catch (err) {
-      if (err.code === 'ENOENT') {
+      if (err.code === 'ENOENT' || err.code === 'ENOTDIR') {
```
Comment (kellyselden) [inline src/get-workspaces-paths.js:59]: `['ENOENT', 'ENOTDIR'].includes(err.code)`?

**Comment by kellyselden** [comment]
a test in every group probably to verify it works in spawn for pnpm and yarn too

**Comment by sbonasu** [comment]
> a test in every group probably to verify it works in spawn for pnpm and yarn too

updated - doesn't need for yarn because yarn only considers it a package if there is package.json with name and version.

File: src/get-workspaces-paths.js (hunk 1 above)
Comment (sbonasu) [reply src/get-workspaces-paths.js]: updated

**Comment by sbonasu** [comment]
> We should have a test for this.

added

**Comment by kellyselden** [comment]
chore(release): 9.1.1

**Pull request merged**